### DISCUSSION TITLE
Backup Date Overview

### DISCUSSION THREAD

**Discussion by flow-motion** [akeeba/panopticon#366] [Ideas]
Hi Niko 
I just asked myself if a "last backup succesfully taken on" - Dashboard Info would be great. And - may be - also a button in the sites panel with: Backup NOW
?
Just an idea

**Comment by nikosdion** [comment]
What constitutes a “successful” backup may have a different meaning for some people than it does for Akeeba Backup. For me, any backup which does not end in an error is successful, even if it failed to upload to remote storage. That, I can report. People, they don't like this. They might consider a backup which they failed to download correctly from storage to be broken. Therefore, I have decided to NEVER report backups as successful or not.

Moreover, as I have explained, I cannot reliably tell you when the last backup was without reloading the list of backups for every site, every minute, which would crash your site and the monitoring server.

As for backup now, it's already there since version 1.0.0.

**Reply by nikosdion** [reply]
I am thinking there is a way to do something like that, see https://github.com/akeeba/panopticon/issues/396

**Comment by flow-motion** [comment]
I am totally with you. Every Backup, that gets a green Hook in Akeeba, is a succesfull backup for the first view. The deeper testing, incl. a rebuild is in the users hand. 
So a column in the overview: last bakup from akeeba succefully taken: jan, 9th would be a win in my eyes. 

Just my thoughts (and in the sites details we find exactly these information - but just in details :) )


have a great time

**Reply by nikosdion** [reply]
As you can see in https://github.com/akeeba/panopticon/issues/396 this is basically what I did. Please do keep in mind that a bit of caching _does_ take place so as not to make the overview page dog crap slow. This means there is a small caveat: you only see this information updated every 15', or if you go into a site's Site Information page and reload its list of backups.

**Reply by flow-motion** [reply]
Hi Nikos - i totally know, it is my fault... but i can´t find it :( - can you send a screenshot?

**Reply by nikosdion** [reply]
<img width="1143" alt="Screenshot 2024-01-25 at 11 53 42" src="https://github.com/akeeba/panopticon/assets/256041/9c5e4684-febd-45f2-960f-88741f80f935">

**Reply by flow-motion** [reply]
okay - that was where i was looking. I also started backups in Akeeba manually... that column is not in my overview :(

**Reply by nikosdion** [reply]
Do you have version 1.1.0?

**Reply by flow-motion** [reply]
yes sir

**Reply by nikosdion** [reply]
Try cleaning the contents of your `tmp` folder.

**Reply by flow-motion** [reply]
In the tmp is only a update.zip  the htaccess / the web-config
I guess these should  be there 
I have a compiled-Templates folder with a lot of .php files. Delete the files?

**Reply by nikosdion** [reply]
Yes, that's what cleaning the contents of a folder means. Delete anything under it, files and subdirectories.

**Reply by flow-motion** [reply]
WOOOOW - i even see fav.icons now from the website. 
That worked! 

Thanks man. 

I am always a little bit afraid of deleting files . ) -

**Reply by nikosdion** [reply]
Cheers!

The `tmp` and `cache` directories' contents are safe to delete. Always. We write our code sensibly, expecting that temporary and cache files may disappear anytime. Not like some Joomla! extensions which put files to keep forever in the cache folder 💀
